src/read.c

```diff
@@ -1953,26 +1953,34 @@ static avifBool avifParseImageGridBox(avifImageGrid * grid,
 
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
 
-static avifBool avifParseToneMappedImageBox(avifGainMapMetadata * metadata, const uint8_t * raw, size_t rawLen, avifDiagnostics * diag)
+static avifBool avifParseToneMappedImageBox(avifGainMapMetadata * metadata,
```

Comment:
I wonder if we should add a changelog item about this change to gain map version check.